ui_testing/elements.py

```diff
@@ -636,7 +636,20 @@
         'suborder_delete': {'method': 'id',
                              'value': 'child_table_delete'},
         'confirm_delete': {'method':'xpath',
-                           'value': '//h2[@id="swal2-title"]'}
+                           'value': '//h2[@id="swal2-title"]'},
+        'save_config': {'method': 'xpath',
+                        'value': '//button[@class="btn btn-primary mb-md-0 mr-md-3"]'},
+        'text_dynamic_field_options': {'method': 'xpath',
```

Comment:
@0xIslamTaha I need help in this field, if other dynamic fields added this field changes